js/exmo.js

```diff
@@ -19,10 +19,13 @@ module.exports = class exmo extends Exchange {
             'has': {
                 'CORS': undefined,
                 'spot': true,
-                'margin': undefined, // has but unimplemented
+                'margin': true,
```

Comment:
I know that this is old, but to switch margin to `true` the methods like `createOrder`, `cancelOrder` ... should all work for margin as well. I'm working on this right now